### PR TITLE
Lack of `info` subcommand for account

### PR DESCRIPTION
**Pull request by r8d8**
Related to #139.

----

- [x] I have read the `CONTRIBUTING.md` and `make ci` passes on my machine.
- [x] I have updated the user manual accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: cmd/nem/main.go [hunk 1]
```diff
@@ -102,6 +106,21 @@ func main() {
 						},
 					},
 				},
+				{
+					Name:   "info",
+					Usage:  "extract info from account",
+					Action: info,
+					Flags: []cli.Flag{
+						cli.BoolFlag{
+							Name:  "address-only, a",
```
Comment (dulanov) [inline cmd/nem/main.go]: Shorthand is good only for most popular options, it's not obvious what is it - `nem account info -a`.

File: cmd/nem/main.go [hunk 2]
```diff
@@ -102,6 +106,21 @@ func main() {
...
+							Usage: "Show address for supplied private key",
+						},
+						cli.BoolFlag{
+							Name:  "public-only, p",
```
Comment (dulanov) [inline cmd/nem/main.go]: Similar remark.

File: cmd/nem/main.go [hunk 3]
```diff
@@ -117,7 +136,7 @@ func generateAction(c *cli.Context) error {
 
 	num := c.Uint("number")
 	for i := uint(0); i < num; i++ {
-		printAccountDetails(ch, keypair.Gen())
+		printAccountDetails(ch, keypair.Gen(nil))
```
Comment (dulanov) [inline cmd/nem/main.go]: Better without `nil` at all, two separate functions - `Gen()` & `FromPrivKey(pr []byte)`.

File: cmd/nem/main.go [hunk 4]
```diff
@@ -200,6 +219,39 @@ func vanityAction(c *cli.Context) error {
 	return nil
 }
 
+func info(c *cli.Context) error {
+	ch, err := chainGlobalOption(c)
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+
+	reader := bufio.NewReader(os.Stdin)
```
Comment (dulanov) [inline cmd/nem/main.go]: Ask user about required private key, write some prompt message.

File: cmd/nem/main.go [hunk 5]
```diff
@@ -200,6 +219,39 @@ func vanityAction(c *cli.Context) error {
...
+	pair := keypair.Gen(bytes.NewReader(pkBytes))
+	if c.Bool("address-only") {
+		printAddress(ch, pair)
+	}
+
+	if c.Bool("public-only") {
+		printPublicKey(pair)
+	}
+
+	if !c.Bool("address-only") && !c.Bool("public-only") {
```
Comment (dulanov) [inline cmd/nem/main.go]: As `else` can be better, or?

File: cmd/nem/main.go [hunk 6]
```diff
@@ -265,7 +317,15 @@ func timeInSeconds(val float64) string {
 
 // printAccountDetails prints account details in pretty user-oriented multi-line format
 func printAccountDetails(chain core.Chain, pair keypair.KeyPair) {
+	printAddress(chain, pair)
+	printPublicKey(pair)
+	fmt.Println("Private key:", hex.EncodeToString(pair.Private))
```
Comment (dulanov) [inline cmd/nem/main.go]: You extracted `public address` and `public key`, is `private key` an exception from this rule?

File: cmd/nem/main.go (hunk 5 above)
Comment (r8d8) [reply cmd/nem/main.go]: Than user can't specify 2 parameters simultaniously -a,  -p

**Pull request merged**